commitlint.config.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
```

Comment:
if we're importing `@commitlint/config-conventional`, why are we pasting the rules manually here?

Reply:
Whoops, I guess I messed up the config a bit. Fixed now